### DISCUSSION TITLE
Having trouble with running brats_18 example

### DISCUSSION THREAD

**Discussion by EvanGalea2** [NVIDIA/NVFlare#1547] [Q&A] [answered]
Hello

I am currently testing the [brats_18 example ](https://github.com/NVIDIA/NVFlare/tree/dev/examples/advanced/brats18)with differential privacy. I have been following the tutorials and although I wasn't able to run the command in the 'prepare local configs' section, I was able to get around it by editing each config file individually

However, when running the setup in the fl_simulator I encounter this error

`  
File "/home/evan/miniconda3/lib/python3.9/importlib/__init__.py", 
line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pt'
`
I've tried running variations of the command to run it, including without using GPU and without any other parameters. from what I can tell, pt seems to be some module, likely from pytorch. I have all the modules installed . I have tried this with both python 3.8 and the latest version (the rest of the setup is just following the readme).

I'm able to run other examples like Cifar10. Could it be an issue with using the python installed in my conda directory?

Also, just to be sure, you guys are able to run the example right?

Apologies if this is an odd question, I'm at my wits end here. Let me know if there's any other info that may be useful in figuring this out.

**Comment by chesterxgchen** [comment]
@ZiyueXu77  can you check if there is some issues ?

**Reply by ZiyueXu77** [reply]
sure

**Comment by chesterxgchen** [comment]
Can you tell us some info, so we can better help you
1) nvfare version
2) python version
3) examples branch ( github branch, dev, main, release branch) ? 
4) os platform

**Reply by EvanGalea2** [reply]
Hi Chesterxgchen, thanks, I should have included this at the start
nvflare 2.3.0rc2
python is tried with both 3.8 and 3.9.5
I am using main branch
System is Ubuntu 20.04

**Reply by chesterxgchen** [reply]
the main branch is the latest stable branch ( which is 2.2.5 release on 2.2 branch), if you are pip install nvflare  or pip install nvflare==2.2.5, you can use main branch examples. 

The 2.3.0rc2 is actually from the unstable dev branch, so the match examples is on dev branch. 
we are likely to branch near the end of month. then the match examples will be on 2.3.0 branch 

Make sure you are using the correct binary and match examples.

**Comment by ZiyueXu77** [comment]
Hi @EvanGalea2 thanks for reaching out! the pt issue is most possibly caused by unrecognized path, you can try adding the brats18/pt to your python path. I will also test from my end when I get a chance and let you know.

**Comment by ZiyueXu77** [comment, marked as answer]
Just did an experiment, and yes, please set PYTHONPATH to include custom files of this example for it to work:

export PYTHONPATH=${PWD}
